app/ui-react/packages/auto-form/stories/Array.stories.tsx

```diff
@@ -30,10 +30,10 @@ stories.add('General Example', () => {
       },
       arrayDefinitionOptions: {
         arrayControlAttributes: {
-          className: 'col-md-1',
+          className: '',
         },
         arrayRowTitleAttributes: {
-          className: 'col-md-2',
+          className: '',
```

Comment:
Feel free to delete this whole `arrayRowTitleAttributes`, if these aren't needed anymore.  Ideally it'd be cool if the array just kinda worked well in like 80% of cases :-)

Reply:
I removed `col-md-2` class and replaced it with one that doesn't change the layout. This should achieve the 80% rule, as the class is not actually required to make anything work. Just a hook in case you need it on the CSS side.